target/linux/mediatek/image/mt7623.mk

```diff
@@ -10,7 +12,8 @@ endef
 TARGET_DEVICES += 7623a-unielec-u7623-02-emmc-512m
 
 define Device/7623n-bananapi-bpi-r2
-  DEVICE_TITLE := MTK7623n BananaPi R2
+  DEVICE_VENDOR := BananaPi
+  DEVICE_MODEL := BPI-R2
```

Comment:
Can this be done similar to the Banana Pi R64?

Reply:
done